Misc/NEWS.d/next/Library/2019-10-16-19-56-51.bpo-38373.86ExWB.rst

```diff
@@ -0,0 +1,2 @@
+:data:`~os.CLD_KILLED` is added for posixmodule.
+(Contributed by Dong-hee Na in :issue:`38493`.)
```

Comment:
`Patch by Dong-hee Na.`

The issue number is already specified in the file name.

Please add a What's New entry for this change (and there the `Contributed by` will be appropriate).